grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -756,6 +756,7 @@ class QueryService implements Status {
                 ' left outer join issue_extra_property p on p.issue_id = i.id ' +
                 ' left outer join user u on p.value = u.user_name ' +
                 ' left outer join funding f on f.project_key = i.project_key ' +
+                ' left outer join consent_collection_link ccl on ccl.consent_key =  i.project_key ' +
```

Comment:
This join looks like a bug to me. In this method, we're looking for Issues, which have a `project_key` column. In the consent collection link table, the join column on issue is also `project_key`. I believe this query should be: 
```
' left outer join consent_collection_link ccl on ccl.project_key =  i.project_key ' +
```
In it's current form, this will only find joins when the issue is a consent group type issue.

Reply:
Another possibility is to join on either case. If the issue is a consent group, **AND** you want all CCLs in the results for the consent group, then you can also join on the consent key:
```
`left outer join consent_collection_link ccl on ccl.project_key = i.project_key or ccl.consent_key = i.project_key`
```